jenkinsfiles/RHEL_Check.groovy

```diff
@@ -193,7 +194,32 @@ pipeline {
                     }
                 }
 
+                stage('Check_code_changes') {
+                    steps {
+                        script {
+                            dir("./src") {
+                                def code_changes = sh(
+                                    script: """
+                                            DOC_STRINGS = $(git diff --name-only main |grep ^documentation)
+                                            STRINGS = $(git diff --name-only origin/main)
+                                            if [[ DOC_STRINGS != STRINGS ]]; then
+                                                exit -1
+                                            fi
+                                            exit 0
+                                            """,
+                                    returnStatus: true, label: "Code_changes")
+                                if (code_changes == 0) {
```

Comment:
This step is not necessary, code_changed  variable defined

Reply:
Right, changed.